### PR TITLE
Update Main Codebase 0.9.34 #CEN-91

### PR DESCRIPTION
**Pull request by obsessed-cake**

### CODE REVIEW

File: frame/merkle-mountain-range/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+[package]
```
Comment (blasrodri) [inline frame/merkle-mountain-range/Cargo.toml:1]: https://github.com/paritytech/substrate/blob/master/frame/beefy-mmr/Cargo.toml

File: frame/merkle-mountain-range/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+name = "pallet-mmr"
```
Comment (blasrodri) [inline frame/merkle-mountain-range/Cargo.toml:2]: what about https://github.com/paritytech/substrate/blob/master/frame/merkle-mountain-range/Cargo.toml

Comment (obsessed-cake) [reply frame/merkle-mountain-range/Cargo.toml:2]: hmm... I tried to find this crate here https://github.com/paritytech/substrate/tree/polkadot-v0.9.34/frame
I'll try to use this version.

**Comment by dzmitry-lahoda** [comment]
@andor0 are you working on it now? do yo need help?

**Comment by obsessed-cake** [comment]
We switched to `polkadot-0.9.36` version.
No longer relevant.

**Pull request closed without merging**